src/soloud/include/soloud.h

```diff
@@ -170,6 +170,18 @@ namespace SoLoud
 		soloudResultFunction mBackendPauseFunc;
 		soloudResultFunction mBackendResumeFunc;
 
+		// Set the callback to call when a voice is ended/stopped
+		void (*_voiceEndedCallback)(unsigned int*) = nullptr;
+		void setVoiceEndedCallback(void (*voiceEndedCallback)(unsigned int*)) {
+			_voiceEndedCallback = voiceEndedCallback;
+		}
```

Comment:
Is this really part of the original `soloud.h`? If not, I'd prefer if this code lived somewhere else.

Reply:
Unfortunately it's not possible

Reply:
As discussed, please add a README file in this directory that explains the custom inclusion of code.